### DISCUSSION TITLE
Stripe Payment API deprecated (no payments possible)

### DISCUSSION THREAD

**Discussion by spralexx** [pretix/pretix#4232] [Q&A] [answered]
### Problem and impact

When using the Stripe payment provider with a newly configured API key it throws the following error when making a payment.

"The Sources API has been deprecated, and we are no longer allowing new integrations. Please reference the following support article to learn how to update your integration: https://support.stripe.com/questions/i-received-a-sources-api-error-how-should-i-handle-it"

According to the docs the "Payment Methods API" replaces the "Sources API" https://docs.stripe.com/payments/payment-methods/transitioning

The new API description can be found here: https://docs.stripe.com/api/payment_methods
 

### Expected behaviour

The Payment is working correctly without error.

### Steps to reproduce

1. Setup an event with Stripe Payment enabled and API Key configured.
2. try to make a payment for this event.

**Comment by raphaelm** [comment, marked as answer]
What pretix version are you running? I believe we have fixed this long ago.

**Reply by spralexx** [reply]
I was using 2023.9.0.dev0 with 2024.6.0.dev0 it is working now. I checked the API version used according to Stipe logs and the version mentioned in https://github.com/pretix/pretix/blob/master/src/pretix/plugins/stripe/payment.py and they were the same so I assumed there were no changes. Unfortunately I must have been on a different branch as on the master everything is fine. 

So sorry for the confusion and thank you for answering.
